### PR TITLE
Making config_id optional

### PR DESCRIPTION
**Pull request by joseims**
the kubejobs is creating a config_id containing an error if the config id isn't in the json

### CODE REVIEW

**Comment by joseims** [comment]
Correct issue #92

File: kubejobs/__init__.py [hunk 1]
```diff
@@ -195,8 +195,10 @@ def update_env_vars(self, data):
         data['env_vars']['REDIS_HOST'] = 'redis-%s' % self.app_id
 
         # inject SCONE_CONFIG_ID in the environment
-        # FIXME: make SCONE_CONFIG_ID optional in submission
-        data['env_vars']['SCONE_CONFIG_ID'] = data['config_id']
+        config_id = data.get('config_id')
+        if (config_id):
```
Comment (matheuscmelo) [inline kubejobs/__init__.py]:
```suggestion
        if config_id is not None:
```
https://docs.quantifiedcode.com/python-anti-patterns/readability/test_for_object_identity_should_be_is_not.html?highlight=none

File: kubejobs/__init__.py [hunk 2]
```diff
@@ -195,8 +195,10 @@ def update_env_vars(self, data):
         data['env_vars']['REDIS_HOST'] = 'redis-%s' % self.app_id
 
         # inject SCONE_CONFIG_ID in the environment
-        # FIXME: make SCONE_CONFIG_ID optional in submission
-        data['env_vars']['SCONE_CONFIG_ID'] = data['config_id']
+        config_id = data.get('config_id')
+        if config_id is not None:
+            data['env_vars']['SCONE_CONFIG_ID'] = config_id
+
```
Comment (joseims) [inline kubejobs/__init__.py]:
```suggestion
```

File: kubejobs/__init__.py [hunk 3]
```diff
@@ -195,8 +195,12 @@ def update_env_vars(self, data):
         data['env_vars']['REDIS_HOST'] = 'redis-%s' % self.app_id
 
         # inject SCONE_CONFIG_ID in the environment
-        # FIXME: make SCONE_CONFIG_ID optional in submission
-        data['env_vars']['SCONE_CONFIG_ID'] = data['config_id']
+        config_id = data.get('config_id')
+        if config_id is not None:
+            KUBEJOBS_LOG.log("You should use pass the config_id variable "
```
Comment (matheuscmelo) [inline kubejobs/__init__.py]:
```suggestion
            KUBEJOBS_LOG.log("You should pass the config_id variable "
```

**Pull request merged**